`proc_macro`, `proc_macro_lib` features need to be removed to work in beta

```
error[E0554]: #[feature] may not be used on the beta release channel
 --> C:\Users\Arnavion\.cargo\registry\src\github.com-1ecc6299db9ec823\derive-new-0.3.0\src\lib.rs:2:1
  |
2 | #![feature(proc_macro, proc_macro_lib)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error
```

--- a/derive-new/src/lib.rs
+++ b/derive-new/src/lib.rs
@@ -1,5 +1,4 @@
 #![crate_type = "proc-macro"]
-#![feature(proc_macro, proc_macro_lib)]
 
 extern crate proc_macro;
 extern crate syn;

